pkg/services/pluginsintegration/renderer/renderer.go

```diff
@@ -22,11 +23,12 @@ import (
 	"github.com/grafana/grafana/pkg/services/pluginsintegration/pipeline"
 	"github.com/grafana/grafana/pkg/services/pluginsintegration/pluginerrs"
 	"github.com/grafana/grafana/pkg/services/rendering"
+	"go.opentelemetry.io/otel/trace"
 )
 
 func ProvideService(cfg *config.PluginManagementCfg, pluginEnvProvider envvars.Provider,
-	registry registry.Service) (*Manager, error) {
-	l, err := createLoader(cfg, pluginEnvProvider, registry)
+	registry registry.Service, tracer tracing.Tracer) (*Manager, error) {
```

Comment:
@wbrowne If I change here wire asks for the `ProvideService` method for the `trace.Tracer` typed tracer but is not the https://github.com/grafana/grafana/blob/main/pkg/infra/tracing/tracing.go#L78 one we want to use?

Reply:
it does implement the `Start` method.

Reply:
Sorry yes that's my mistake!